src/checkstyle/Checker.hx

```diff
@@ -246,7 +250,23 @@ class Checker {
 		}
 	}
 
-	@SuppressWarnings("checkstyle:Dynamic")
+	function checkForExclude(moduleName:String):Bool {
+		if (excludes == null) return false;
+		var excludesForCheck:Array<String> = excludes.get(moduleName);
+		if (excludesForCheck == null || excludesForCheck.length == 0) return false;
+
+		var cls = file.name.substring(0, file.name.indexOf(".hx"));
```

Comment:
Not sure if the `indexOf(".hx")` is a good idea here. We may need to add support for other extensions in the future, maybe with a file filter regex argument. For example, OpenFL uses `.hxp` for project files (much less popular than the .xml format though): http://www.openfl.org/documentation/projects/project-files/hxp-format/